pkg/cloud/packet/util.go

```diff
@@ -21,9 +21,9 @@ import (
 )
 
 const (
-	machineUIDTag = "cluster-api-provider-packet:machine-uid"
-	clusterIDTag  = "cluster-api-provider-packet:cluster-id"
-	namespaceTag  = "cluster-api-provider-packet:namespace"
+	machineUIDTag = "capp:machine-uid"
```

Comment:
This change is a breaking change that will need to be called out with upgrade steps (along with the ramifications).

Presumably, changing the tags that are used in determining what to manage will cause the service to disown existing resources (without cleanup) and provision new resources. Is this for EIPs and servers? Control plane nodes too/

Reply:
I recognize that this was needed to fit more restrictive tag lengths in EM API resources.